### ISSUE TITLE
beta - oncoprint - after adding clinical tracks, "add clinical track" text disappears

### ISSUE THREAD

**Issue by tmazor** [cBioPortal/cbioportal#4476]
Before adding a clinical track:
![image](https://user-images.githubusercontent.com/10508276/43006775-95b89966-8c04-11e8-89bf-d748d2005632.png)

After adding a clinical track:
![image](https://user-images.githubusercontent.com/10508276/43006789-9e41230a-8c04-11e8-801e-393595d83de5.png)

**Comment by adamabeshouse** [comment]
redirecting to @fedde-s , this is an issue with react-select-checked

**Comment by fedde-s** [comment]
Hm… this looks exactly like the issue I resolved in https://github.com/thehyve/react-select-checked/pull/19. Are you using release v0.1.9 of `react-select-checked`?

**Comment by fedde-s** [comment]
Is this about the code in the current `release-x.x.x` branch, or just `rc`?

**Comment by fedde-s** [comment]
Ah, the milestone suggests it's about the release candidate.

**Comment by rnugraha** [comment]
We are investigating this. We've tried in a non-cbioportal playground and it's working we're still figuring out the root cause.

**Comment by adamabeshouse** [comment]
@rnugraha thanks. perhaps it has something to do with mobx?

**Comment by fedde-s** [comment]
> perhaps it has something to do with mobx?

If so, one particular thing that might be worth checking, is whether any complex props are being observed *outside* of the non-MobX component and passed in as plain JavaScript data structures. Particularly passing observable arrays to non-MobX libraries has broken things for me before, and the solution was `.slice()`. But we might have discussed that already.
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by rnugraha** [comment]
@adamabeshouse doesn't look like it. I've solved the placeholder issue, but there are also another issue which is unchecking is not working that seems related with the onChange event in the cbioportal code itself. Will ask you in slack.

Temporarily just like in the cbioportal-frontend master branch, the version v0.1.4 and v0.1.5 seems to work fine, maybe its better to use this version till the problem has been solved.
